spanner/spanner_snippets/snippet.go

```diff
@@ -253,7 +253,12 @@ func queryUsingIndex(ctx context.Context, w io.Writer, client *spanner.Client) e
 	stmt := spanner.Statement{
 		SQL: `SELECT AlbumId, AlbumTitle, MarketingBudget
 			FROM Albums@{FORCE_INDEX=AlbumsByAlbumTitle}
-			WHERE AlbumTitle >= 'Aardvark' AND AlbumTitle < 'Goo'`}
+			WHERE AlbumTitle >= @start_title AND AlbumTitle < @end_title`,
+		Params: map[string]interface{}{
+			"start_title": "Aardvark",
```

Comment:
I dont think its too bad. If it really concerns you, I am ok with not having default value at all and have the user always specify the value.